apps/panier/.env

```
@@ -0,0 +1,18 @@
+# URL de l'API Supabase
+NEXT_PUBLIC_SUPABASE_URL=http://127.0.0.1:54321
+# Clé publique d'accès à l'API Supabase
+NEXT_PUBLIC_SUPABASE_ANON_KEY=encrypted:BDrCITn/ZEsjywzRhX9HFsk6ZqeELiM6N+DJo6uWVtrnLjLrWcqj5lb3wgI+giwxCLQ1PYg3qdrP7xMJZr6UQkscRcaOXBx4eqFIS4tUQjD+ok9Fqvycz1lH9UJDJA2EB5qgQaUmT5uFZrkc9XVB+BoSnEDR43jrMpTQEhOnJoZ12aBUqoFW9jGOLbAxXzB9wZJ6W4kM1ArXmnk7oF2kC7VdW6EdfHZ+yjyi7JrRrm9fRMe/vhftYO3p9bNurJAx5BRJ1P0WXI/Gp7bBoPClftkpeHlKSZhIbLekMPqqM+pZnBrWUWWd0DdpM+maXBZhPpDwwgcC2uQ1ig==
+# URL de l'API PostHog
+POSTHOG_HOST=""
+# Key de l'API PostHog
+POSTHOG_KEY=""
+# Identifiant d'accès à l'API Crisp
+NEXT_PUBLIC_CRISP_WEBSITE_ID=""
+# Identifiant d'accès à l'API Axeptio
+NEXT_PUBLIC_AXEPTIO_ID="axeptio-id"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)